.tests/test-workflow/config/cancer_wes/config.yaml.jinja2

```diff
@@ -0,0 +1,63 @@
+static_data_config:
+  reference:
+    path: {{ params["reference"] }}
+
+step_config:
+  ngs_mapping:
+    tools:
+      dna: [bwa]    # Required if DNA analysis; otherwise, leave empty. Example: 'bwa'.
+    bwa:
+      path_index: {{ params["bwa_index"] }}
+      mask_duplicates: false
+      memory_bam_sort: 2G
+      num_threads_bam_sort: 2
+      num_threads_bam_view: 2
+      num_threads_align: 2
+    ngs_chew_fingerprint:
+      enabled: false
+    target_coverage_report:
+      path_target_interval_list_mapping:
+      - name: MedExome_hg19_empirical_targets
+        pattern: MedExome
+        path: ../resources/Exome-MedExome.chr12.bed
+
+
+  somatic_variant_calling:
+    tools: [mutect2]
+    mutect2:
+      ngs_mapping: ../ngs_mapping
+      extra_arguments: [] # [ "--genotype-germline-sites true", "--genotype-pon-sites true" ]
+      window_length: 300000000
+      keep_tmpdir: onerror
+      job_mult_time: 5
+
+
+  somatic_variant_annotation:
+    path_somatic_variant_calling: ../somatic_variant_calling  # REQUIRED
+    tools: ["vep"]
+    vep:
+      cache_dir: {{ params["vep_cache"] }}
+      assembly: GRCh37
+
+
+  somatic_variant_filtration:
+    path_somatic_variant: ../somatic_variant_annotation  #  instead of somatic_variant_calling
+    path_ngs_mapping: ../ngs_mapping
+    filter_list:
+    - dkfz: {}
+    - bcftools:
+        exclude: FORMAT/DP[1]<=50 | AD[1:1]<5 | AD[1:1]/(AD[1:0]+AD[1:1])<0.025
+    - bcftools:
```

Comment:
I would remove this filter: it can only be justified when the tumor have been sequenced from FFPE material